src/lightning_lite/plugins/precision/precision.py

```diff
@@ -34,6 +36,20 @@ def forward_context(self) -> Generator[None, None, None]:
         """A contextmanager for managing model forward/training_step/evaluation_step/predict_step."""
         yield
 
+    def convert_input(self, data: Tensor) -> Tensor:
+        """Convert model inputs (forward) to the floating point precision type of this plugin.
+
+        This is a no-op for tensors that are not of floating-point type or already have the desired type.
+        """
+        return _convert_fp_tensor(data, torch.float32)
+
+    def convert_output(self, data: Tensor) -> Tensor:
```

Comment:
what if the input was not of default type? Will we still convert the output to default type or should that rather be converted to the inputs type?

Reply:
Good point Justus. Maybe we should not convert the output at all, or do the conversion back to input as you said? For example, using double precision, we would downgrade to lower precision (32) but maybe user needs the double for further loss calculations. 

In both these cases, we wouldn't need a convert_output() method and would handle it directly in the lite forward wrapper.

Reply:
In any case, right now the implementation is in parity to what was before. But no saying the previous way was more correct.

Reply:
The safest thing IMO would be to convert to the original data precision. Otherwise not do this at all.

Can you create an issue about this for tracking?

Reply:
Created issue #14859

Removed the convert_output method. We can introduce it later should we have a decision. Right now, I'll just leave the output as it was handled previously.